src/koans/15_destructuring.clj

```diff
@@ -8,37 +8,38 @@
 
 (meditations
   "Destructuring is an arbiter: it breaks up arguments"
-  (= __ ((fn [[a b]] (str b a))
+  (= ":bar:foo" ((fn [[a b]] (str b a))
          [:foo :bar]))
 
   "Whether in function definitions"
   (= (str "An Oxford comma list of apples, "
           "oranges, "
           "and pears.")
-     ((fn [[a b c]] __)
+     ((fn [[a b c]] (str "An Oxford comma list of " a ", " b ", and " c "."))
       ["apples" "oranges" "pears"]))
 
   "Or in let expressions"
   (= "Rich Hickey aka The Clojurer aka Go Time aka Lambda Guru"
      (let [[first-name last-name & aliases]
            (list "Rich" "Hickey" "The Clojurer" "Go Time" "Lambda Guru")]
-       __))
+        (str first-name " " last-name (reduce str (map #(str " aka " %) aliases)))))
```

Comment:
You could also use [apply](https://clojuredocs.org/clojure.core/apply) instead of reduce.
Also this is a nice example for using [interpose](https://clojuredocs.org/clojure.core/interpose). For example:
```
(str first-name " " last-name " aka " (apply str (interpose " aka " aliases)))
```